Tests/Admin/AdminTest.php

```diff
@@ -1523,19 +1524,11 @@ public function testGetSubject()
         $admin = new PostAdmin('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle:PostAdmin');
         $admin->setModelManager($modelManager);
 
-        $admin->setRequest(new Request(array('id' => 'azerty')));
```

Comment:
The result comes from a mock, so it will not be null, but always `$entity`. And that is tested line 1532